bids/layout/models.py

```diff
@@ -698,12 +698,12 @@ def _init_on_load(self):
         if self._dtype == 'json':
             self.value = json.loads(self._value)
             self.dtype = 'json'
+        elif self._dtype == 'bool':
```

Comment:
I think the `_dtype` will be the actual datatype. So `bool` instead of `'bool'`.

Reply:
`_dtype` is the database field, `dtype` is the Python type.

https://github.com/bids-standard/pybids/blob/8fb92fa748f518404a0f22d3e856f9abb5f21ada/bids/layout/models.py#L668